### PR TITLE
Traslate YDB concepts

### PR DESCRIPTION
**Pull request by Pseudolukian**
### Changelog category
* Documentation (changelog entry is not required)

### CODE REVIEW

File: ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md [hunk 1]
```diff
@@ -282,7 +282,7 @@ You will see that the cluster was initialized successfully when the cluster init
 
 ## Create a database {#create-db}
 
-To work with tables, you need to create at least one database and run a process (or processes) to serve this database (dynamic nodes):
+To work with [row](../../concepts/datamodel/table.md#row-oriented-tables) and [column](../../concepts/datamodel/table.md#column-oriented-tables) oriented tables, you need to create at least one database and run a process (or processes) to serve this database (dynamic nodes):
```
Comment (anton-bobkov) [inline ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md:285]: To work with [row-oriented](../../concepts/datamodel/table.md#row-oriented-tables) and [column-oriented](../../concepts/datamodel/table.md#column-oriented-tables) tables

**Review by anton-bobkov — COMMENTED** [review]
Мелкие стилистические правки

File: ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md [hunk 2]
```diff
@@ -440,14 +440,25 @@ When running the account creation and group assignment commands, the {{ ydb-shor
 
 1. Install the {{ ydb-short-name }} CLI as described in the [documentation](../../reference/ydb-cli/install.md).
 
-1. Create a `test_table`:
+1. Create a testing row (`test_row_table`) or column (`test_column_table`) oriented table:
```
[the remark this answers is not in the document]
Comment (Pseudolukian) [reply ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md:443]: Учёл

File: ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md [hunk 3]
```diff
@@ -440,14 +440,25 @@ When running the account creation and group assignment commands, the {{ ydb-shor
```
[the remark this answers is not in the document]
Comment (Pseudolukian) [reply ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md:440]: Учёл

File: ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md (hunk 1 above)
Comment (Pseudolukian) [reply ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md:285]: Учёл

File: ydb/docs/en/core/concepts/_includes/index/how_it_works.md [hunk 4]
```diff
@@ -2,54 +2,64 @@
...
 
-### Table
+### Table {#table}
 
 ![Table](https://storage.yandexcloud.net/ydb-www-prod-site-assets/howitworks/table.png)
 
-YDB provides users with a well-known abstraction: tables. In YDB tables must contain a primary key, the data is physically sorted by the primary key. Tables are automatically sharded by primary key ranges by size or load. Each primary key range of a table is handled by a specific tablet called *data shard*.
+{{ ydb-short-name }} provides users with the well-known abstraction — tables. In {{ ydb-short-name }}, there are two main types of tables:
+* [Row tables](../../datamodel/table.md#row-tables) are designed for OLTP workloads.
+* [Column tables](../../datamodel/table.md#column-tables) are designed for OLAP workloads.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/index/how_it_works.md:27]:
```suggestion
{{ ydb-short-name }} provides users with a well-known abstraction — tables. In {{ ydb-short-name }}, there are two main types of tables:

* [Row-oriented tables](../../datamodel/table.md#row-tables) are designed for OLTP workloads.
* [Column-oriented tables](../../datamodel/table.md#column-tables) are designed for OLAP workloads.
```

File: ydb/docs/en/core/concepts/_includes/index/how_it_works.md [hunk 5]
```diff
@@ -2,54 +2,64 @@
...
 
-#### Split by load
+Logically, from the user's point of view, both types of tables look the same. The main difference between row tables and column tables lies in the way data is stored. In row tables, the values of all columns in each row are stored together, while in column tables — on the contrary, each column is stored separately, so cells related to different rows are stored next to each other.
+
+Regardless of its type, each table must contain a primary key. Column tables can use only  `NOT NULL` columns as primary keys. Data in tables is physically sorted by the primary key. Partitioning works differently in row and column tables:
+* Row tables are automatically partitioned by the ranges of primary keys depending on the volume of data.
+* Column tables are partitioned by the hash of the partitioning columns.
+
+Each range of primary keys in a table is processed by a specific [tablet](../../cluster/common_scheme_ydb.md#tablets), called a data shard for row tables and a column shard for column tables.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/index/how_it_works.md:35]:
```suggestion
Logically, from the user's perspective, both types of tables look the same. The main difference between row-oriented tables and column-oriented tables is in how the data is physically stored. In row-oriented tables, the values of all columns in each row are stored together. In contrast, in column-oriented tables, each column is stored separately, meaning cells related to different rows are stored next to each other.

Regardless of type, each table must have a primary key. Column tables can only use `NOT NULL` columns as primary keys. Table data is physically sorted by the primary key. 

Partitioning works differently in row and column tables:

* Row-oriented tables are automatically partitioned by primary key ranges, depending on the data volume.
* Column-oriented tables are partitioned by the hash of the partitioning columns.

Each partition of a table is processed by a specific [tablet](../../glossary.md#tablets), called a [data shard(../../glossary.md#datashard) for row-oriented tables and a [column shard](../../glossary.md#columnshard) for column-oriented tables.
```

File: ydb/docs/en/core/concepts/_includes/index/intro.md [hunk 6]
```diff
@@ -17,11 +17,11 @@ description: "YDB is a horizontally scalable distributed fault-tolerant DBMS. YD
 
 To interact with {{ ydb-short-name }}, you can use the [{{ ydb-short-name }} CLI](../../../reference/ydb-cli/index.md) and [SDK](../../../reference/ydb-sdk/index.md) fo C++, C#, Go, Java, Node.js, PHP, Python, and Rust.
 
-{{ ydb-short-name }} supports a relational [data model](../../../concepts/datamodel/table.md) and manages [tables](../../datamodel/table.md) with a predefined schema. To make it easier to organize tables, directories can be created like in the file system. In addition to tables, {{ ydb-short-name }} supports [topics](../../topic.md) as an entity for storing unstructured messages and delivering them to multiple subscribers.
+{{ ydb-short-name }} supports a relational [data model](../../../concepts/datamodel/table.md) and manages [row oriented](../../datamodel/table.md#row-oriented-tables) and  [column oriented](../../datamodel/table.md#column-oriented-tables) tables with a predefined schema. To make it easier to organize tables, directories can be created like in the file system. In addition to tables, {{ ydb-short-name }} supports [topics](../../topic.md) as an entity for storing unstructured messages and delivering them to multiple subscribers.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/index/intro.md:20]:
```suggestion
{{ ydb-short-name }} supports a relational [data model](../../../concepts/datamodel/table.md) and manages [row-oriented](../../datamodel/table.md#row-oriented-tables) and  [column-oriented](../../datamodel/table.md#column-oriented-tables) tables with a predefined schema. To make it easier to organize tables, directories can be created like in the file system. In addition to tables, {{ ydb-short-name }} supports [topics](../../topic.md) as an entity for storing unstructured messages and delivering them to multiple subscribers.
```

File: ydb/docs/en/core/concepts/_includes/index/intro.md [hunk 7]
```diff
@@ -17,11 +17,11 @@ description: "YDB is a horizontally scalable distributed fault-tolerant DBMS. YD
...
 
 Database commands are mainly written in YQL, an SQL dialect. This gives the user a powerful and already familiar way to interact with the database.
 
-{{ ydb-short-name }} supports high-performance distributed [ACID](https://en.wikipedia.org/wiki/ACID_(computer_science)) transactions that may affect multiple records in different tables. It provides the serializable isolation level, which is the strictest transaction isolation. You can also reduce the level of isolation to raise performance.
+{{ ydb-short-name }} supports high-performance distributed [ACID](https://en.wikipedia.org/wiki/ACID_(computer_science)) transactions that may affect multiple records of the same type in different tables. It provides the serializable isolation level, which is the strictest transaction isolation. You can also reduce the level of isolation to raise performance.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/index/intro.md:24]:
```suggestion
{{ ydb-short-name }} supports high-performance distributed [ACID](https://en.wikipedia.org/wiki/ACID_(computer_science)) transactions that may affect multiple records in different tables. It provides the serializable isolation level, which is the strictest transaction isolation. You can also reduce the level of isolation to raise performance.
```

File: ydb/docs/en/core/concepts/_includes/scan_query.md [hunk 8]
```diff
@@ -24,6 +24,7 @@ Scan queries cannot currently be considered an effective solution for running OL
 * There is no optimization for point reads or reading small ranges of data.
 * The SDK doesn't support automatic retry.
 
+For handling OLAP workloads in {{ ydb-short-name }}, there is a specialized type of table — [column](../datamodel/table.md#column-oriented-tables) tables. They store the data of each column separately from other columns. As a result, only those columns that are directly involved in the query are read when executing a query.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/scan_query.md:27]:
```suggestion
For handling OLAP workloads in {{ ydb-short-name }}, there is a specialized type of table — [column-oriented](../datamodel/table.md#column-oriented-tables) tables. These tables store the data of each column separately from other columns. This allows only the columns directly involved in the query to be read during execution.
```

File: ydb/docs/en/core/concepts/_includes/ttl.md [hunk 9]
```diff
@@ -4,7 +4,7 @@ This section describes how the TTL mechanism works and what its limits are. It a
 
 ## How it works {#how-it-works}
 
-{{ ydb-short-name }} lets you specify a table column (TTL column), whose values set the lifetime of items. TTL automatically deletes the item from your table once the specified number of seconds passes after the time set in the TTL column.
+{{ ydb-short-name }} lets you specify a TTL column in a [row-oriented](../datamodel/table.md#row-oriented-tables) and [column-oriented](../datamodel/table.md#column-oriented-tables) table. Values in TTL columns set the lifetime of items.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/ttl.md:7]:
```suggestion
{{ ydb-short-name }} allows you to specify a TTL column in both [row-oriented](../datamodel/table.md#row-oriented-tables) and [column-oriented](../datamodel/table.md#column-oriented-tables) tables. Values in TTL columns determine the lifetime of table rows.
```

File: ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md [hunk 10]
```diff
@@ -436,18 +436,29 @@ In the command examples above, `<node.ydb.tech>` is the FQDN of the server runni
...
+## Start using the created database {#try-first-db}
 
 1. Install the {{ ydb-short-name }} CLI as described in the [documentation](../../reference/ydb-cli/install.md).
 
-1. Create a `test_table`:
+1. Create a test row (`test_row_table`) or column (`test_column_table`) oriented table:
 
+{% list tabs %}
+
+- Creating a row oriented table
```
Comment (blinkov) [inline ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md:447]:
```suggestion
- Creating a row-oriented table

```

File: ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md [hunk 11]
```diff
@@ -436,18 +436,29 @@ In the command examples above, `<node.ydb.tech>` is the FQDN of the server runni
...
+   ```bash
+   ydb --ca-file ca.crt -e grpcs://<node.ydb.tech>:2136 -d /Root/testdb --user root \
+      yql -s 'CREATE TABLE `testdir/test_row_table` (id Uint64, title Utf8, PRIMARY KEY (id));'
+   ```
+
+- Creating a column oriented table
```
Comment (blinkov) [inline ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md:453]:
```suggestion
- Creating a column-oriented table

```

**Review by blinkov — CHANGES_REQUESTED** [review]

**Comment by blinkov** [comment]
also, "This branch has conflicts that must be resolved"

File: ydb/docs/en/core/concepts/_includes/index/how_it_works.md [hunk 12]
```diff
@@ -2,54 +2,68 @@
...
+{{ ydb-short-name }} provides users with a well-known abstraction — tables. In {{ ydb-short-name }}, there are two main types of tables:
 
-#### Split by load
+* [Row-oriented tables](../../datamodel/table.md#row-tables) are designed for OLTP workloads.
+* [Column-oriented tables](../../datamodel/table.md#column-tables) are designed for OLAP workloads.
+
+Logically, from the user's perspective, both types of tables look the same. The main difference between row-oriented tables and column-oriented tables is in how the data is physically stored. In row-oriented tables, the values of all columns in each row are stored together. In contrast, in column-oriented tables, each column is stored separately, meaning cells related to different rows are stored next to each other.
+
+Regardless of type, each table must have a primary key. Column tables can only use `NOT NULL` columns as primary keys. Table data is physically sorted by the primary key.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/index/how_it_works.md:32]:
```suggestion
Logically, from the user’s perspective, both types of tables look the same. The main difference between row-oriented and column-oriented tables lies in how the data is physically stored. In row-oriented tables, the values of all columns in each row are stored together. In contrast, in column-oriented tables, each column is stored separately, meaning that cells from different rows are stored next to each other within the same column.

Regardless of the type, each table must have a primary key. Column-oriented tables can only have `NOT NULL` columns in primary keys. Table data is physically sorted by the primary key.
```

File: ydb/docs/en/core/concepts/_includes/index/how_it_works.md [hunk 13]
```diff
@@ -2,54 +2,68 @@
...
+
+Partitioning works differently in row and column tables:
+
+* Row-oriented tables are automatically partitioned by primary key ranges, depending on the data volume.
+* Column-oriented tables are partitioned by the hash of the partitioning columns.
+
+Each partition of a table is processed by a specific [tablet](../../glossary.md#tablets), called a [data shard(../../glossary.md#datashard) for row-oriented tables and a [column shard](../../glossary.md#columnshard) for column-oriented tables.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/index/how_it_works.md:39]:
```suggestion
Partitioning works differently in row-oriented and column-oriented tables:

* Row-oriented tables are automatically partitioned by primary key ranges, depending on the data volume.
* Column-oriented tables are partitioned by the hash of the partitioning columns.

Each partition of a table is processed by a specific [tablet](../../glossary.md#tablets), called a [data shard](../../glossary.md#datashard) for row-oriented tables and a [column shard](../../glossary.md#columnshard) for column-oriented tables.
```

File: ydb/docs/en/core/concepts/_includes/index/intro.md [hunk 14]
```diff
@@ -12,7 +12,7 @@
 
 To interact with {{ ydb-short-name }}, you can use the [{{ ydb-short-name }} CLI](../../../reference/ydb-cli/index.md) and [SDK](../../../reference/ydb-sdk/index.md) fo C++, C#, Go, Java, Node.js, PHP, Python, and Rust.
 
-{{ ydb-short-name }} supports a relational [data model](../../../concepts/datamodel/table.md) and manages [tables](../../datamodel/table.md) with a predefined schema. To make it easier to organize tables, directories can be created like in the file system. In addition to tables, {{ ydb-short-name }} supports [topics](../../topic.md) as an entity for storing unstructured messages and delivering them to multiple subscribers.
+{{ ydb-short-name }} supports a relational [data model](../../../concepts/datamodel/table.md) and manages [row-oriented](../../datamodel/table.md#row-oriented-tables) and  [column-oriented](../../datamodel/table.md#column-oriented-tables) tables with a predefined schema. To make it easier to organize tables, directories can be created like in the file system. In addition to tables, {{ ydb-short-name }} supports [topics](../../topic.md) as an entity for storing unstructured messages and delivering them to multiple subscribers.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/index/intro.md:15]:
```suggestion
{{ ydb-short-name }} supports a relational [data model](../../../concepts/datamodel/table.md) and manages [row-oriented](../../datamodel/table.md#row-oriented-tables) and [column-oriented](../../datamodel/table.md#column-oriented-tables) tables with a predefined schema. To make it easier to organize tables, directories can be created like in the file system. In addition to tables, {{ ydb-short-name }} supports [topics](../../topic.md) as an entity for storing unstructured messages and delivering them to multiple subscribers.
```

File: ydb/docs/en/core/concepts/_includes/index/when_use.md [hunk 15]
```diff
@@ -9,3 +9,4 @@
 * In systems with unpredictable or seasonally fluctuating load (you can add/reduce computing resources on request and/or in serverless mode).
 * In high-load systems that shard load across relational DB instances.
 * When developing a new product with no reliable load forecast or with an expected high load beyond the capabilities of conventional relational databases.
+* In projects where simultaneous handling of transactional and analytical workloads is required.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/index/when_use.md:12]:
```suggestion
* In projects where the simultaneous handling of transactional and analytical workloads is required.
```

File: ydb/docs/en/core/concepts/_includes/ttl.md [hunk 16]
```diff
@@ -4,7 +4,7 @@ This section describes how the TTL mechanism works and what its limits are. It a
 
 ## How it works {#how-it-works}
 
-{{ ydb-short-name }} lets you specify a table column (TTL column), whose values set the lifetime of items. TTL automatically deletes the item from your table once the specified number of seconds passes after the time set in the TTL column.
+{{ ydb-short-name }} allows you to specify a TTL column in both [row-oriented](../datamodel/table.md#row-oriented-tables) and [column-oriented](../datamodel/table.md#column-oriented-tables) tables. Values in TTL columns determine the lifetime of table rows.
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/_includes/ttl.md:7]:
```suggestion
{{ ydb-short-name }} allows you to specify a TTL column in both [row-oriented](../datamodel/table.md#row-oriented-tables) and [column-oriented](../datamodel/table.md#column-oriented-tables) tables. Values in TTL columns determine the table rows lifetime.
```

**Review by blinkov — DISMISSED** [review]

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**